lib/ex_wubook/query.ex

```diff
@@ -67,7 +67,7 @@ defmodule ExWubook.Query do
   Decode query result
   """
   def decode_response(%{success: true, response: %{status: 200, body: body}} = payload) do
-    with {:ok, decoded_response} <- XMLRPC.decode(body) do
+    with {:ok, decoded_response} <- body |> HtmlEntities.decode() |> XMLRPC.decode() do
```

Comment:
Please, create private function called `decode` and move this pipe into this function.
This make with condition much more clear.

Reply:
Changed